packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -153,6 +150,28 @@ const NuovaDelega = () => {
       setLoadAllEntities(true);
     }
   };
+  // done for complexity
+
+  const handleFilterAutocomplete = (options: Array<Party>, state: { inputValue: string }) => {
```

Comment:
Same in SpecialContacts.tsx above, no needs to use state parameter. Use directly the variable.

Reply:
do it